turn/__init__.py

```diff
@@ -27,19 +27,38 @@ def do_request(self, data=None):
             data=json.dumps(data) if data is not None else None,
         )
 
+    def get_error(self, response):
```

Comment:
This might as well return the details, since it looks like that's the only part that the calling code uses.

Reply:
There is (sadly?) at least one case that the code actually matters (from what I saw most of the codes Turn says they send are actually not sent, but it's the only way to distinguish if a 404 is because the user didn't exist or because the template didn't exist.